### PR TITLE
Update plant_sciences.md

### PR DESCRIPTION
**Pull request by ErwanLeFloch**

### CODE REVIEW

**Review by lauportell — APPROVED** [review]

File: pages/your_domain/plant_sciences.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Here are described the mandatory, recommended and optional metadata fields for d
 
 #### Checklists, ontologies and file formats
 * Sharing plant genotyping data files involves the use of the Variant Call Format (VCF) standard.  
-* Findability and reusability of VCF files depends on the supplied metadata and in particular with MIAPPE compliant biological material description: the [plant VCF metadata FAIR Cookbook](INSERT LINK) helps you on that topic.
+* Findability and reusability of VCF files depends on the supplied metadata and in particular with MIAPPE compliant biological material description: the [plant VCF metadata FAIR Cookbook (under development)](INSERT LINK) helps you on that topic.
```
Comment (bedroesb) [inline pages/your_domain/plant_sciences.md:132]:
```suggestion
* Findability and reusability of VCF files depends on the supplied metadata and in particular with MIAPPE compliant biological material description: the plant VCF metadata FAIR Cookbook (under development) helps you on that topic.
```

**Comment by bedroesb** [comment]
@ErwanLeFloch I would personally not add a placeholder for a link if there is none. We will add it when the recipe is online, if that is ok for you

**Comment by ErwanLeFloch** [comment]
@bedroesb sure, it is ok for me

**Comment by bedroesb** [comment]
Looking forward to the recipe!

**Pull request merged**